chromadb/api/__init__.py

```diff
@@ -59,11 +60,11 @@ def create_collection(
         """Creates a new collection in the database
 
         Args:
```

Comment:
Why are we removing the type hints here?

Reply:
Those aren't the type hints, those are just text in the method docstring. I removed them because:

1. They are redundant with the real type hints above.
2. They were incorrect.
3. They are difficult to keep in sync with the real types.
4. These aren't even the user-facing APIs (which are in the `Collection` model).

Reply:
I am using the word type hint liberally. But understood.